src/view/actions/sendEvent.jsx

```diff
@@ -60,22 +55,22 @@ const getDecisionScopesFromFormState = values => {
 const getInitialDecisionScopesData = decisionScopes => {
   if (Array.isArray(decisionScopes)) {
     return {
-      decisionsInputMethod: decisionScopesOptions.CONSTANT,
-      decisionScopesArray: decisionScopes,
+      decisionsInputMethod: CONSTANT,
+      decisionScopesArray: decisionScopes.map(scope => ({ value: scope })),
```

Comment:
One downside of React Hook Form is that when dealing with editing arrays of items, the items must be objects (and not just strings, like we had). This is a current limitation of React Hook Formik because the object is used to store a key that [helps with performance when re-rendering lists of items](https://reactjs.org/docs/lists-and-keys.html#keys).